plugin/unimpaired.vim

```diff
@@ -77,6 +77,7 @@ call s:MapNextFamily('t', 't', 'trewind')
 
 function! s:entries(path) abort
   let path = substitute(a:path,'[\\/]$','','')
+  let path = substitute(path, '\([\|\$\)', '[\1]', 'g')
```

Comment:
Try `substitute(path, '[[$*]', '[&]', 'g')` to make it a bit cleaner. I threw `*` in there also for good measure.

Reply:
Thank you for quick advice. I updated the code and confirmed it works!
https://github.com/tpope/vim-unimpaired/pull/232/commits/dd00e8ac2f4e1907652ffb844dbf7d92e6d30698